### ISSUE TITLE
[Bug] Missing sensors

### ISSUE THREAD

**Issue by Vaskivskyi** [Vaskivskyi/ha-asusrouter#780]
### The problem

@sezlony, hey. Let's troubleshoot your missing sensors. Please, list the sensors which are missing. Do you use `router` mode in the integration or any other (e.g. `ap`, `aimesh node`, `media bridge`)

### Your device model

RT-AC1200G+

### Firmware type

Stock

### Firmware version

3.0.0.4.382.52272

### Integration version with the issue

---

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core

**Comment by sezlony** [comment]
ok, let me see:

yes, I'm using plain router mode, nothing fancy

missing all the speed monitoring sensors, like up-download via bridge, wan etc.

since they became useless with the upgrade I got in panic and my first move was to delete them hoping the integration will rediscover them, but...

**Comment by Vaskivskyi** [comment]
Can you please check, whether all the other sensors and other entities are present? Some of them might be `disabled` in HA if you haven't yet enabled them

Including, CPU, RAM, LAN / WAN ports, WLAN (Wireless) and GWLAN and so on

This would help find out whether the issue is with querying or processing one endpoint or some more general

**Comment by sezlony** [comment]
I have 54 entities present, a lot of them is disabled by the integration, these are only device trackers and I didn't enable them as I considered them unnecessary

other critical entities like those you listed are present and working fine (as far as I can tell), I'm using their data

only thing disabled intensionally by me is AiMesh, although I can't recall where and how I managed to do that

**Comment by Vaskivskyi** [comment]
Can you please share the diagnostics provided by AsusRouter? You can get the file by clicking `Download diagnostics`. You can either share it here or send to Yevhenii@Vaskivskyi.com

![image](https://github.com/Vaskivskyi/ha-asusrouter/assets/6554415/317a6c7f-ca22-46b5-950f-73fb86b936d7)

**Comment by sezlony** [comment]
here you go
[config_entry-asusrouter-f6cc682d3d23cc78a6f5c083c25fba89.json.txt](https://github.com/Vaskivskyi/ha-asusrouter/files/14105835/config_entry-asusrouter-f6cc682d3d23cc78a6f5c083c25fba89.json.txt)

**Comment by Rembock** [comment]
Also having this issue, can share diagnostics and other details if needed

**Comment by Vaskivskyi** [comment]
Still trying to figure it out or reproduce...

Yes, please, @Rembock, diagnostics would help. Or at least your device model and FW version

**Comment by lutzee** [comment]
I'm just seeing this issue, but this is my first time setting up this integration so I never got to see the speed sensors in the first place. 

**Device model**
ZenWiFi AC Mini (CD6)

**Firmware type**
Stock

**Firmware version**
3.0.0.4.386_49693-gecfd731

**Integration version with the issue**
0.28.0

**Method of the integration installation**
HACS

**What version and type of Home Assistant installation do you use**
HAOS

**Is there anything useful in the logs?**
Don't know

**Diagnostics information**
[config_entry-asusrouter-061ad4e9feb33132b97aa3043785bb5a (1).json](https://github.com/Vaskivskyi/ha-asusrouter/files/14201985/config_entry-asusrouter-061ad4e9feb33132b97aa3043785bb5a.1.json)

**Additional information**
I see that my router is in the expected to be working but not tested category, so if there is anything you'd like me to check in that regard I'm happy to help. I've also attached a diagnostics log, but I don't see much in there that's particularly revealing in there.

[home-assistant_asusrouter_2024-02-07T21-48-50.224Z.log](https://github.com/Vaskivskyi/ha-asusrouter/files/14201987/home-assistant_asusrouter_2024-02-07T21-48-50.224Z.log)

**Comment by sezlony** [comment]
no luck with 0.29.0 either, the formerly (e.g. 0.22.0) present network speed sensors are still gone, no change noticed in comparison with 0.28.0

**Comment by Rembock** [comment]
@sezlony that's true, but that's because there's nothing about this issue mentioned in the release changelog :)

**Comment by sezlony** [comment]
yeah, I know and you could've been right, unless you knew that Vaskivskyi himself sent a direct mail adivsing: "Please, check with version 0.29.0. If the issue persists, please..."

:)

**Comment by peetereczek** [comment]
Issue still exists, devices which are trackable are 'disabled by integration' during startup

**Comment by rac146** [comment]
also seeing same issue - looking specifically for the speed test sensors!

ET12/Router mode.. can give any other info if that will help..

**Comment by dev-null-undefined** [comment]
Same issue here running 0.34.0 on RT-AC1200 with  Firmware: 3.0.0.4.380.10931_gfed566f 
No speed sensors

**Comment by dev-null-undefined** [comment]
What would be some first steps that I could try debug why they are missing?
For now I created statistics helper from the total downloaded GB, but that isn't very precise
